src/internal/types.ts

```diff
@@ -276,3 +276,99 @@ export type ValueFromNotification<T> = T extends { kind: 'N' | 'E' | 'C' }
 export type Falsy = null | undefined | false | 0 | -0 | 0n | '';
```

Comment:
@benlesh Completely off topic, but shouldn't this include `NaN`?

Reply:
Yeah, probably.

Reply:
`Nan` is not a type; it's a value, AFAICT; `number` is the type.